src/annotator/config/settings.js

```diff
@@ -13,10 +13,14 @@ import { urlFromLinkTag } from './url-from-link-tag';
  * @prop {string} clientUrl
  * @prop {string} sidebarAppUrl
  * @prop {string} notebookAppUrl
- 
- * @prop {(name: string, options?: object) => (string|null)} hostPageSetting
+ * @prop {(name: string) => unknown} hostPageSetting
```

Comment:
The declared return type was wrong. The function returns whatever value the client configuration has for a particular key, and we expect other value types (eg. `openSidebar` may be a boolean).